Update README.md

RamonRB opened the pull request:
I'm not entirely sure if I did this correctly in Git, as I'm not too familiar with it, but I added a rule for review. Content of the rule:

**302.** Repeal of rule 204. Voting AGAINST a rule does not incur any penalties or reward any benefits. To promote additional rules (and thus playing the game), voting against a rule should not be rewarded. Gaining points by voting AGAINST is unconducive to the game and should be discouraged.

RamonRB committed:
Update README.md

akvadrako commented:
I like the simplicity of it, though it doesn't take effect yet - since proposals still need unanimity. I would vote yes, though based on my experience from last round, I will wait to vote until the proposal is finalized - maybe you'll get some suggestions from others.

rknegjens commented:
I agree that repealing 204 will lead to a better game. I will wait with voting until everyone has shown sign of life.

roarandrock commented:
If the goal is to promote additional rules, why are you getting rid of Rule 204 and not Rule 206?

204. If and when rule-changes can be adopted without unanimity, the players who vote against winning proposals shall receive 10 points each.

206. When a proposed rule-change is defeated, the player who proposed it loses 10 points

rknegjens commented:
> If the goal is to promote additional rules, why are you getting rid of Rule 204 and not Rule 206?
> 
>     204. If and when rule-changes can be adopted without unanimity, the players who vote against winning proposals shall receive 10 points each.
> 
>     206. When a proposed rule-change is defeated, the player who proposed it loses 10 points

By rule 103 only one rule can be repealed at a time. I think 204 does more damage to adding new rules, as I expect voters are more likely to consider a boost to their own score versus a hit to the proposers (unless the proposer is close to 100).

rknegjens commented on README.md:
```diff
@@ -95,3 +95,5 @@ g
 
 This rule takes precedence over every other rule determining the winner. 
+
+**302.** Repeal of rule 204. Voting AGAINST a rule does not incur any penalties or reward any benefits. To promote additional rules (and thus playing the game), voting against a rule should not be rewarded. Gaining points by voting AGAINST is unconducive to the game and should be discouraged.
```
The first sentence is sufficient for defining the rule, while the next two sentences explain the spirit of the rule. 

**Question to everyone:** do we find such explanations useful to have in a rule or should we keep rules as brief as possible?

RamonRB replied:
As you said, it is to explain the spirit of the rule, so that when in the future there is discussion on whether it should repealed we know what the intent of it was so to better determine its value.

As to why repealing 204 rather than 206? Frankly 206 did not occur to me, but that is a good one for another round.

akvadrako commented:
Perhaps it would be better to include the explanation as a note. Rule 103 came with a note, which I moved to the notes.md page. The reason is that if the additional sentences have legal force, there may be unintended consequences.

Yours are probably okay since they say should, which is just suggestive. But perhaps if one is implicitly awarded by voting AGAINST due to the text of a rule, the rule or vote would be illegal.

rknegjens commented:
Yea I would prefer explanations as a note. Otherwise I'm happy with this rule and will vote for it.

RamonRB committed:
Note on rule 302

Voting AGAINST a rule does not incur any penalties or reward any benefits. To promote additional rules (and thus playing the game), voting against a rule should not be rewarded. Gaining points by voting AGAINST is unconducive to the game and should be discouraged.

RamonRB commented:
I changed the content of readme.md and added the notes in the change comments. Was that the idea?

akvadrako commented:
Nice. I will update the points...

akvadrako commented:
And I will also remove rule change rule 204 into just "repealed".
